buddyboss/buddyboss-platform
# PROD-662 Fixed #3237 minified jquery.atwho.min.js file

Just minified jquery.atwho.min.js file which was missed in the fix #2239 which is closed now, and I reopened in #3237

### All Submissions:






### Changes proposed in this Pull Request:



Fixes #3237 .

### How to test the changes in this Pull Request:

1. Download and enable the Korean Language in Windows and switch to korean
2. Goto Forums and try to create a new discussion
3. See the error in console

### Proof Screenshots or Video

https://www.loom.com/share/eaad637f3ed04881b4e90fb4e6737c72



### Other information:




### Changelog entry

> Enter a summary of all changes on this Pull Request. This will appear in the changelog if accepted.
